stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -379,11 +379,6 @@ function(_add_target_variant_link_flags)
     swift_android_cxx_libraries_for_arch(${LFLAGS_ARCH} cxx_link_libraries)
     list(APPEND link_libraries ${cxx_link_libraries})
 
-    # link against the ICU libraries
-    list(APPEND link_libraries
-      ${SWIFT_ANDROID_${LFLAGS_ARCH}_ICU_I18N}
-      ${SWIFT_ANDROID_${LFLAGS_ARCH}_ICU_UC})
```

Comment:
Missed this tweak from the rebase yesterday, because it was moved to a different file upstream, added it back now.